Tests/DefaultsTests/DefaultsAnySeriliazableTests.swift

```diff
@@ -466,4 +466,10 @@ final class DefaultsAnySerializableTests: XCTestCase {
 
 		waitForExpectations(timeout: 10)
 	}
+
+    func testWrongCast() {
+        let value = Defaults.AnySerializable(false)
+        XCTAssertEqual(value.get(Bool.self), false)
+        XCTAssertNil(value.get(String.self))
+    }
```

Comment:
Same as the above comment, and please use `XCTAssertFalse` instead of `XCTAssertEqual` as linter suggests.

Reply:
I initially used that, but since `get` returns on optional, I'd have to provide a default value. That feels less clean.